web/l10n/en.json

```diff
@@ -46,9 +46,11 @@
     "sharedAlias": "Alias",
     "sharedComputedAttribute": "Computed Attribute",
     "sharedComputedAttributes": "Computed Attributes",
+    "sharedCheckComputedAttribute": "Check Computed Attribute",
     "sharedExpression": "Expression",
     "sharedDeviceDistance": "Device Distance",
     "sharedDevice": "Device",
+    "sharedSelectDevice": "Select Device",
```

Comment:
Can we just reuse `sharedDevice` here? I want to avoid adding unnecessary new string for translation.